### PR TITLE
BS5 work

### LINKED ISSUE #8: Use other source for fonts
Same as https://cynkra.com .

### LINKED ISSUE #12: Links for package authors
Example: If Christoph is a package author, his name in the "Authors" list should be a hyperlink pointing to https://www.cynkra.com/about.html .

How do they do it in https://github.com/tidyverse/tidytemplate?

### LINKED ISSUE #16: use BS5
in particular it'd add a search bar

### PR DESCRIPTION
**Pull request by maelle**
Fix #16 
Fix #12 
Fix #8

TODOs

- [ ] Get svg icon
- [ ] Check with other visual identity criteria
- [ ] Add a dropdown menu to see how it looks like / should be tweaked

image

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -2,28 +2,19 @@ url: https://github.com/cynkra/cynkratemplate
...
 template:
   package: cynkratemplate
-  default_assets: false
 
 destination: docs
 
 navbar:
   structure:
     left: [news]
-    right: [reference, articles, projects, consulting, github]
```
Comment (maelle) [inline _pkgdown.yml:12]: projects was a dead link

File: inst/pkgdown/_pkgdown.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    link-color: "#184FA9"
+    base_font: "frutiger-light,sans-serif"
+    code_font: {google: "Fira Mono"}
+
+footer:
+  structure:
+    left: developed_by
+    right: cynkra
+  components:
+    cynkra: <p>Sponsored by <a class="cynkra-logo-font-footer" href="https://www.cynkra.com">cynkra</a><a class="cynkra-logo-footer" href="https://www.cynkra.com">&#9632;</a>. Visit us at <a href="https://www.cynkra.com/">www.cynkra.com</a>.</p>
```
Comment (maelle) [inline inst/pkgdown/_pkgdown.yml:18]: it seems easier than having a template HTML, so IMHO it's ok not to have the package name here. Thoughts?

File: inst/pkgdown/_pkgdown.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+authors:
```
Comment (maelle) [inline inst/pkgdown/_pkgdown.yml:20]: @pat-s this is the list I meant

**Comment by maelle** [comment]
Note that for code I chose Fira Mono which does not include ligatures because I read it's better for beginners to not show ligatures (whereas for a font one uses in one's local IDE, there aren't such limitations... except when teaching I suppose).

**Comment by krlmlr** [comment]
Thanks. Merging now to build the pkgdown site for preview, can you please tackle remaining issues in a branch prefixed with `docs-*` ?

File: inst/pkgdown/_pkgdown.yml (hunk 3 above)
Comment (pat-s) [reply inst/pkgdown/_pkgdown.yml:20]: Thanks, I created a CU task.

**Pull request merged**